docs/self-managed/operational-guides/backup-restore/webapps-backup.md

```diff
@@ -1,41 +1,51 @@
 ---
-id: operate-tasklist-backup
-title: Backup and restore Operate and Tasklist data
-description: "How to perform a backup and restore of Operate and Tasklist data."
+id: webapps-backup
+title: Backup and restore Webapps (Operate, Tasklist and Optimize) data
+description: "How to perform a backup and restore of Webapps (Operate, Tasklist and Optimize) data."
 keywords: ["backup", "backups"]
 ---
 
 import Tabs from "@theme/Tabs";
 import TabItem from "@theme/TabItem";
 
-:::warning breaking changes!
-As of the Camunda 8.6 release, the `/actuator` endpoints (including `/backups`) now default to port 9600. Ensure your `management.server.port` configuration parameter is correctly set before continuing.
+:::warning breaking changes
+As of the Camunda 8.7 release, the `/actuator` endpoints for backups have been moved to `/actuator/backupHistory`. The old `/actuator/backups` endpoint is still active only if the applications are deployed standalone (i.e. each application in its own process)
 :::
 
-Operate stores its data over multiple indices in Elasticsearch. Backup of Operate data includes several
-Elasticsearch snapshots containing sets of Operate indices. Each backup is identified by `backupId`. For example, a backup with an ID of `123` may contain the following Elasticsearch snapshots:
+:::note
+In this page we will refer to "webapps" as Operate, Tasklist and Optimize. Depending on your deployment configuration, you may not have all of them deployed. If that is the case, you can safely ignore the configurations for those applications.
+:::
+
+Webapps store their data over multiple indices in Elasticsearch. Backup of Webapps data includes several
+Elasticsearch snapshots containing sets of different indices. Each backup is identified by `backupId`. For example, a backup with an ID of `123` may contain the following Elasticsearch snapshots:
 
 ```
-camunda_operate_123_8.1.0_part_1_of_6
-camunda_operate_123_8.1.0_part_2_of_6
-camunda_operate_123_8.1.0_part_3_of_6
-camunda_operate_123_8.1.0_part_4_of_6
-camunda_operate_123_8.1.0_part_5_of_6
-camunda_operate_123_8.1.0_part_6_of_6
+camunda_webapps_123_8.1.0_part_1_of_6
+camunda_webapps_123_8.1.0_part_2_of_6
+camunda_webapps_123_8.1.0_part_3_of_6
+camunda_webapps_123_8.1.0_part_4_of_6
+camunda_webapps_123_8.1.0_part_5_of_6
+camunda_webapps_123_8.1.0_part_6_of_6
 ```
 
-Operate provides an API to perform a backup and manage backups (list, check state, delete). Restore a backup using the standard Elasticsearch API.
+All webapps provides the same API to perform a backup and manage backups (list, check state, delete). Restore a backup using the standard Elasticsearch API.
 
 :::note
 The backup API can be reached via the Actuator management port, which by default is the same as application HTTP port (and in turn defaults to 9600). The port may be reconfigured with the help of `management.server.port` configuration parameter.
```

Comment:
```suggestion
, which by default is the same as application HTTP port (and in turn defaults to 9600). 
```
I know you haven't written this, but this confuses me. "same as application HTTP port" isn;t this the 8080 or 80?